src/Symfony/Component/Yaml/Tests/Fixtures/unindentedCollections.yml

```diff
@@ -60,3 +60,23 @@ yaml: |
         foo: bar
 php: |
     array('collection' => array('key' => array('a', 'b', 'c'), 'foo' => 'bar'))
+---
+test: Shortcut Key after unindented collection
+brief: >
+    Key/value after unindented collection
+yaml: |
+    collection:
+    -  key: foo
```

Comment:
I would use one space here. Otherwise, this example were the same as the one below.